src/javis_viewer.jl

```diff
@@ -127,12 +127,18 @@ function _javis_viewer(
     # Create a textbox
     tbox = GtkReactive.textbox(Int; signal = signal(slide))
 
+    # fontCss =  """GtkButton {
+    #             font-size: 1000px;
+    #         }"""
+
+    # provider = GtkCssProviderLeaf(data = fontCss)
     # Button for going forward through animation
-    forward = GtkButton("==>")
+    # forward = GtkButton("\U021D2")
+    forward = GtkButton("Next Frame")
 
     # Button for going backward through animation
-    backward = GtkButton("<==")
-
+    # backward = GtkButton("\U021D0")
+    backward = GtkButton("Previous Frame")
```

Comment:
Personally, I would like to keep the viewer as language agnostic as possible. Sure, our docs are in English, but my hope is to translate those docs to other languages someday. Not a huge fan of the English...

Could something else work?

Reply:
I just had a short look at the css example and I have no clue how Gtk works :smile: 
Gotta go to bed now. Hope you can figure it out. Thanks @Sov-trotter 

I would just leave it as `==>` if we can't figure out font-size or icon.

Reply:
I did try css and I do feel that it may come in handy sometime in the future when it is desired to extend the functionality of the viewer. :)